### ISSUE TITLE
Outdated UI screenshots in the documentation

### ISSUE THREAD

**Issue by tomasz1986** [syncthing/docs#550]
Some of the screenshots showing the Syncthing's UI in the documentation are very outdated, e.g. from v0.10.x. The UI itself is not that much different, but there are at least some missing, renamed, or removed elements. I think that it would be a good idea to replace them with their counterparts from a more recent version of Syncthing.

I would like to update them, but first I just want to ask, which Web browser would be preferred to use for this?

**Comment by calmh** [comment]
Please do. I don't the browser matters greatly, minimizing the amount of chrome (i.e., non-Syncthing UI, not avoiding Chrome the browser) is probably good. Might even crop the screenshots to skip the browser UI entirely and avoid this issue in the future.

**Comment by tomasz1986** [comment]
Yeah, I was thinking about cropping the browser elements out completely, leaving only the Syncthing UI. There are still differences in font rendering, etc. between each browser, hence the question. I will probably use Chromium then.

**Comment by tomasz1986** [comment]
The issue is still open, because there are yet a few outdated screenshots around, but I just wanted to say that the screenshots from the Docs seem to be used for images presented when searching for Syncthing on Google, and someone also added one to Wikipedia.

- https://www.google.com/search?q=syncthing
  ![image](https://user-images.githubusercontent.com/5626656/107316461-feca8780-6adb-11eb-816b-9278087b31c0.png)
  
- https://en.wikipedia.org/wiki/Syncthing
  ![image](https://user-images.githubusercontent.com/5626656/107316514-1a359280-6adc-11eb-9049-94cf146d705f.png)
  
I guess that this gives keeping them up-to-date a little bit more importance :grin:.
